### ISSUE TITLE
Support for react-scripts 3.4.0 (webpack 4.41.5)

### ISSUE THREAD

**Issue by Patrik-Lundqvist** [DocSpring/craco-less#33]
The latest npm release of craco-less (1.15.0) is not compatible with the current version of CRA (3.4.0)

[react-script now requires webpack 4.41.5](https://github.com/facebook/create-react-app/blob/2030ee162df0ca8f19efedcf0447e02613e2f53d/packages/react-scripts/package.json#L79)

**Comment by tux-tn** [comment]
Hello Patrick.
```craco-less``` is now using ```webpack```as dev dependency to prevent such version mismatch conflicts.
I am using 1.15.0 with react-scripts 3.4.0 right now and i don't have any problem running craco. Can you give more info about the issue your are encountering ? Any error message?

**Comment by Patrik-Lundqvist** [comment]
Moving webpack as a dev dependency was done after the 1.15.0 release, no?

npm [still shows](https://www.npmjs.com/package/craco-less?activeTab=dependencies) webpack 4.41.2 as a regular dependency

```
├─┬ craco-less@1.15.0
  └── webpack@4.41.2
```

The issue I'm encountering is in the CRA preflight check (complains about detecting webpack 4.41.2 )

**Comment by tux-tn** [comment]
The commit is not included in 1.15.0 release, my bad.
The needed change (updating to webpack 4.14.5) has already been pushed to the repo but since only @ndbroadbent can push new version to npm we need to wait for a 1.16.0 release (I already notified Nathan of the changes).

You have two temporary workarounds while waiting for a new release:

- Adding ```SKIP_PREFLIGHT_CHECK=true``` to your .env file to prevent the preflight checks from running.
- Switching temporary to yarn, i was not having your issue because yarn's dependencies resolution was installing the correct version.

And Sorry for the inconvenience!

**Comment by Patrik-Lundqvist** [comment]
Yeah, fortunately there are easy workarounds; just thought I'd give a heads-up. No worries at all, thanks for the work you do here! 👍

**Comment by tux-tn** [comment]
craco-less 1.16.0 has been released and should fix the cra preflight check warning. 
Thank you @Patrik-Lundqvist for giving a heads-up about the issue

**Comment by pybuche** [comment]
Hi ! 

Did anyone had this solved ?
I'm using both `craco-less` and `craco-antd` and I can't make the production build to start (I get the same issue as here https://github.com/DocSpring/craco-less/issues/30), but with `react-scripts` 3.4.1 & webpack `4.41.5`...

The weird thing is: the first build worked, but when I rebuild it, it's not working anymore :/
I'm currently trying to adapt my versions to stabilize my build but can't manage to do it correctly apparently 🤔 

Here's my `package.json`:
```
{
    "name": "staycation-admin",
    "version": "1.0.0",
    "private": true,
    "description": "Staycation client UI",
    "author": "Staycation",
    "license": "ISC",
    "engines": {
        "node": "~12"
    },
    "scripts": {
        "start": "craco start",
        "build": "craco build",
        "test": "craco test",
        "lint:js": "eslint --color ./src",
        "lint:js:fix": "yarn lint:js --fix",
        "lint:sass": "sass-lint -v -q",
        "lint:sass:fix": "sass-lint-auto-fix",
        "lint": "yarn lint:sass && yarn lint:js",
        "type-check": "tsc --pretty",
        "style-check": "yarn type-check && yarn lint",
        "full-check": "yarn style-check && yarn test"
    },
    "dependencies": {
        "@ant-design/icons": "^4.0.6",
        "@craco/craco": "5.5.0",
        "@mars/heroku-js-runtime-env": "3.0.2",
        "@sentry/browser": "^5.15.4",
        "@sentry/cli": "^1.52.3",
        "@types/jest": "^25.2.1",
        "@types/node": "^13.13.2",
        "@types/react": "^16.9.34",
        "@types/react-dom": "^16.9.6",
        "antd": "4.1.4",
        "axios": "0.19.2",
        "connected-react-router": "6.8.0",
        "craco-antd": "1.14.1",
        "craco-less": "1.16.0",
        "date-fns": "2.0.0-alpha.25",
        "geolib": "^3.2.1",
        "history": "4.10.1",
        "jwt-decode": "2.2.0",
        "lodash": "^4.17.15",
        "moment": "2.24.0",
        "node-sass": "^4.13.1",
        "prop-types": "^15.7.2",
        "react": "16.13.1",
        "react-dom": "16.13.1",
        "react-photo-gallery": "8.0.0",
        "react-redux": "7.2.0",
        "react-router-dom": "5.1.2",
        "react-scripts": "3.3.1",
        "react-sortable-hoc": "1.11.0",
        "redux": "4.0.5",
        "redux-devtools-extension": "2.13.8",
        "redux-form": "8.3.5",
        "redux-saga": "1.1.3",
        "slug": "2.1.1",
        "typescript": "^3.8.3",
        "url-search-params": "^1.1.0"
    },
    "devDependencies": {
        "@types/lodash": "^4.14.149",
        "@types/react-redux": "^7.1.7",
        "@typescript-eslint/eslint-plugin": "^2.21.0",
        "@typescript-eslint/parser": "^2.21.0",
        "eslint-config-airbnb-typescript": "^5.0.0",
        "eslint-import-resolver-typescript": "^2.0.0",
        "eslint-import-resolver-webpack": "^0.9.0",
        "eslint-plugin-import": "^2.18.2",
        "eslint-plugin-jest": "21.15.1",
        "eslint-plugin-jsx-a11y": "^6.2.3",
        "eslint-plugin-lodash": "^6.0.0",
        "eslint-plugin-react": "^7.14.3",
        "eslint-plugin-react-hooks": "^1.7.0",
        "react-test-renderer": "16.8.6",
        "sass-lint-auto-fix": "^0.21.0"
    },
    "browserslist": [
        ">0.2%",
        "not dead",
        "not ie <= 11",
        "not op_mini all"
    ]
}
```
